CREDITS

```diff
@@ -1,6 +0,0 @@
-
```

Comment:
IMO you should keep the credits, because those new designs and icons has been created by community-design-team, and we should add their credits. I think adding [this](https://gitlab.com/fedora/design/community-design-team/issues/-/issues/146/) and listing the creators will be enough.

Reply:
Resolved